outputs.tf

```diff
@@ -1,13 +1,29 @@
-# TODO: insert outputs here.
+output "private_endpoints" {
+  description = "A map of private endpoints. The map key is the supplied input to var.private_endpoints. The map value is the entire azurerm_private_endpoint resource."
+  value       = azurerm_private_endpoint.this
+}
 
-# Module owners should include the full resource via a 'resource' output
-# https://azure.github.io/Azure-Verified-Modules/specs/terraform/#id-tffr2---category-outputs---additional-terraform-outputs
 output "resource" {
-  value       = azurerm_this_module_resource.this
   description = "This is the full output for the resource."
+  value       = azurerm_mssql_server.this
```

Comment:
We should not output the whole resource any more, see updated spec. Only output specific computed outputs. DO not output anything that is also an input (apart from name)